README.md

```diff
@@ -29,51 +35,65 @@ Create an open-access side-view vessel image dataset using images from the M2 sy
-### How to get Dataset
-- Option 1: via AWS S3 bucket  
-    All datasets are located in Orcasound's S3 bucket: `s3://visual-sandbox/orca-eye-aye/data/`  
-Open data archive could be accessed via e.g. --
-`aws --no-sign-request s3 sync s3://visual-sandbox/orca-eye-aye/`.  
-- Option 2: via Roboflow Universe  
-  At Roboflow Universe, you can access our entire collection of labeled datasets, including ongoing projects, here: https://universe.roboflow.com/cuize/
 
+### Phase 2: Train models with the labeled data
 
+For the initial release, we created two automated real-time vessel object detection systems. The first is for detecting vessels in the image. The second is for vessel classification.    
 
-### Phase 2: Train model with labeled data
-Create two automated real-time vessel object detection systems. One is for detecting the existence of vessels in the image and is also used as a filter for better-quality data, and the other is for vessel classification.    
+For detecting the existence of vessels, the quality of the training data is not required to be very high, but the outlines still need to be discernible. 
 
+![Blank diagram (1)](https://github.com/orcasound/orca-eye-aye/assets/47680801/9682150b-9de8-4c5e-92e0-313ba7a0c3a5)
 
+We used transfer learning with pre-trained Yolo5-Yolo8 models. The code used to process the labled data sets from the Roboflow annotation app, train the model, and assess its performance is shared via [the VesselDetection_train iPython notebook](VesselDetection_train.ipynb) in this reposistory under the MIT software license.
+ 
+The model is stored alongside the training data within the Visual Sandbox bucket and is freely shared under a Responsible AI Licence (RAIL). The [Orca Aye Eye RAIL](Orca-Eye-Aye-RAIL.md) was customized from boilerplate acquired via the [Responsible AI license generator](https://www.licenses.ai/rail-license-generator). 
 
+## How to access the labeled dataset, models, and weights
 
-For detecting the existence of vessels, the quality of the training data is not required to be very high, but the outlines still need to be discernible. Since there is no open access dataset for vessels, our project extends to task 2: compile a high-quality dataset for vessels ( looking for a license for the dataset). This dataset is a compilation and classification of the accumulated data from the M2 system and will be crucial for the future training of deep learning models for vessel detection and classification.    
+Within the 'visual-sandbox/orca-eye-aye/' S3 directory, there are three folders:
 
-![Blank diagram (1)](https://github.com/orcasound/orca-eye-aye/assets/47680801/9682150b-9de8-4c5e-92e0-313ba7a0c3a5)
+1. *data* holds our labeled dataset.
+2. *model* stores our most recent model(s).
+3. *results* presents inferences based on image data from Orcasound Lab.
 
+Model weights are stored in this repo within the `weights` directory.
```

Comment:
Given that we are using yolov8, and not making changes to the structure of the model, "model weights" and "model" mean the same thing.  I would change this sentence to: "Models are also stored in this repo within the `weights` directory."